projects/laji-ui/src/lib/typeahead/positioning/positioning.service.ts

```diff
@@ -69,6 +72,9 @@ export class PositioningService {
             return;
           }
 
+          // We could call placementService.update here, but I don't think it's necessary
+          // The old ngx-bootstrap positioning logic has been entirely replaced
+          return;
           this.positionElements
```

Comment:
I think it would be clearer to comment out the whole section that is not used anymore or remove it altogether as it can always be found in the history if it's needed. I'm also wondering if this really is unnecessary code, I tried to resize the window while the typeahead was open and it didn't change place, should it?